app/controllers/submit.js

```diff
@@ -1008,33 +1008,18 @@ export default Controller.extend(Analytics, BasicsValidations, NodeActionsMixin,
             this.toggleProperty('shareButtonDisabled');
             model.set('provider', this.get('currentProvider'));
 
-            let submitAction = null;
-            if (this.get('moderationType')) {
-                submitAction = this.get('store').createRecord('review-action', {
-                    actionTrigger: 'submit',
-                    target: this.get('model'),
-                });
-                this.set('submitAction', submitAction);
-            } else {
+            const isModerated = this.get('moderationType');
+            if (!isModerated) {
                 model.set('isPublished', true);
             }
             node.set('public', true);
-
-            this.set('node', node);
             this.set('model', model);
+            this.set('node', node);
 
-            let saveChanges = null;
-            if (submitAction) {
-                saveChanges = model.save()
-                    .then(this._saveNode.bind(this))
```

Comment:
_saveNode doesn't show up anywhere else now afaict, can you remove the extra function:
https://github.com/baylee-d/ember-osf-preprints/blob/236594f38f2306f449bf1209c18d2191023e7ea5/app/controllers/submit.js#L1307